### PR TITLE
Remove Rocket from our dependencies

### PR DESCRIPTION
**Pull request by timothyfroehlich**
It's not being used by anything and is causing cargo-deny issues

### CODE REVIEW

File: crates/figma_import/README.md [hunk 1]
```diff
@@ -8,25 +9,28 @@ The Figma API provides access to documents, images, vectors, and interactions th
...
+* `image_context` manages all of the images in the document. Figma's API takes a big set of "image references" and then returns a document containing URLs for each of those image references. `image_context` then fetches the images, and stores the network bytes and dimensions. It can encode and decode its list of fetched images and sizes -- this is how the list of previously fetched images is sent to the client and reused for the next request (avoiding having `image_context` fetch images that the client already has).
+  * `svg` is used by `document` and `image_context` to rasterize complex vectors to rasters (storing the resulting rasters as images in `image_context`). Currently vectors get rasterized at 1x, 2x, and 3x sizes so that the DesignCompose client can choose the best image based on the device's pixel density. Figma provides an API to rasterize vectors, which we don't use because it clips the edges off of rasterized vectors.
```
Comment (iamralpht) [inline crates/figma_import/README.md:32]: not sure why this is in the diff, but it's also wrong (the client rasterizes vectors in all client impls now).

**Review by iamralpht — APPROVED** [review]
good to remove rocket, nice to clean up the README or remove the bits that didn't change from this diff.

File: crates/figma_import/README.md (hunk 1 above)
Comment (timothyfroehlich) [reply crates/figma_import/README.md:32]: I think it got run through my formatter. I'll raise an issue to get the readme updated.

**Comment by timothyfroehlich** [comment]
I'll leave the formatted changes. They match the format that we use for the rest of the docs.

**Pull request merged**